src/components/common/YourCompany/OfficeAddresses.vue

```diff
@@ -506,10 +519,9 @@ export default class OfficeAddresses extends Mixins(CommonMixin) {
    */
   private setLocalProperties (): void {
     if (this.isCorrectionFiling || this.isAlterationFiling) {
-      if (this.getOfficeAddresses?.registeredOffice) {
-        this.mailingAddress = { ...this.getOfficeAddresses.registeredOffice.mailingAddress }
-        this.deliveryAddress = { ...this.getOfficeAddresses.registeredOffice.deliveryAddress }
-      }
+      // assign registered office addresses (may be {})
+      this.mailingAddress = { ...this.getOfficeAddresses?.registeredOffice?.mailingAddress }
+      this.deliveryAddress = { ...this.getOfficeAddresses?.registeredOffice?.deliveryAddress }
```

Comment:
Extra conditional chaining operators are needed due to conversions maybe not having addresses. Also below.